common/include/pcl/impl/point_types.hpp

```diff
@@ -1538,19 +1382,23 @@ namespace pcl
   PCL_EXPORTS std::ostream& operator << (std::ostream& os, const ReferenceFrame& p);
   struct EIGEN_ALIGN16 ReferenceFrame : public _ReferenceFrame
   {
-    inline ReferenceFrame (const _ReferenceFrame &p)
+    inline constexpr ReferenceFrame (const _ReferenceFrame &p) :
+      ReferenceFrame{p.rf}
     {
-      std::copy_n(p.rf, 9, rf);
+      //std::copy_n(p.rf, 9, rf); // this algorithm is constexpr starting from C++20
     }
 
-    inline ReferenceFrame ()
+    inline constexpr ReferenceFrame () :
+      _ReferenceFrame{ {{0.0f}} }
```

Comment:
Let's add 8 others 0 here?

Reply:
do you suggest it as a readability improvement? this is technically correct according to aggregate initialization rules